docs/sources/flow/getting-started/migrating-from-prometheus.md

```diff
@@ -187,3 +187,28 @@ prometheus.remote_write "default" {
   }
 }
 ```
+
+## Limitations
```

Comment:
Added this because it's a good idea like promtail has and it is referenced by the new static migration guide